packages/components/src/toolbar-button/index.js

```diff
@@ -35,6 +35,7 @@ function ToolbarButton( {
 
 	const button = (
 		<Button
+			aria-label={ title }
 			icon={ icon }
 			label={ title }
```

Comment:
Can it be set in the Button block if it's aware of the label? The double label is a bit confusing.

Reply:
Yup, you are right, the good catch here! 
WIll prepare the fix for it!